src/features/transfer/useFeeQuotes.ts

```
@@ -1,52 +1,75 @@
-import { TokenAmount, WarpCore } from '@hyperlane-xyz/sdk';
+import { Token, WarpCore, WarpCoreFeeEstimate } from '@hyperlane-xyz/sdk';
 import { HexString } from '@hyperlane-xyz/utils';
 import { getAccountAddressAndPubKey, useAccounts } from '@hyperlane-xyz/widgets';
 import { useQuery } from '@tanstack/react-query';
 import { logger } from '../../utils/logger';
 import { useMultiProvider } from '../chains/hooks';
-import { getTokenByIndex, useWarpCore } from '../tokens/hooks';
+import { useWarpCore } from '../tokens/hooks';
+import { getTransferToken } from './TransferTokenForm';
```

ESLint

[error] Dependency cycle detected.

(import/no-cycle)